package.json

```diff
@@ -46,12 +49,14 @@
   "devDependencies": {
     "event-stream": "^4.0.0",
     "jshint": "^2.5.6",
+    "nodemon": "^2.0.4",
     "pelias-mock-logger": "^1.1.0",
     "precommit-hook": "^3.0.0",
     "proxyquire": "^2.0.0",
     "tap-dot": "^2.0.0",
     "tape": "^4.2.2",
-    "temp": "^0.9.0"
+    "temp": "^0.9.0",
+    "winston": "^3.3.3"
```

Comment:
Please remove both of these as I don't see how they're used in this PR. Winston is already included through [pelias-logger](https://github.com/pelias/logger/), and I don't think we want to prescribe a tool like `nodemon`.